cmd/kube-controller-manager/app/plugins.go

```diff
@@ -91,12 +89,14 @@ func NewVolumeProvisioner(cloud cloudprovider.Interface, config componentconfig.
 	switch {
 	case cloud == nil && config.EnableHostPathProvisioning:
 		return getProvisionablePluginFromVolumePlugins(host_path.ProbeVolumePlugins(volume.VolumeConfig{}))
-	case cloud != nil && aws.ProviderName == cloud.ProviderName():
-		return getProvisionablePluginFromVolumePlugins(aws_ebs.ProbeVolumePlugins())
-	case cloud != nil && gce.ProviderName == cloud.ProviderName():
-		return getProvisionablePluginFromVolumePlugins(gce_pd.ProbeVolumePlugins())
-	case cloud != nil && openstack.ProviderName == cloud.ProviderName():
-		return getProvisionablePluginFromVolumePlugins(cinder.ProbeVolumePlugins())
+	case config.EnableNetworkStorageProvisioning:
```

Comment:
@jsafrane yeah, some logic has to be developed here first to decide whether its gluster /ceph or some other network storage provisioner .  Second logic has to be there to enable one cloud provisioner and one network storage provisioner at the same time. Is there any work going on in this direction? if that is the case I may need to wait for it. If there is no plan as such, Is it fine to continue on this and getting this done in this manner ? please let me know your thoughts on this, so that I can proceed accordingly.